src/pages/Filing/FilingApp/FileSubmission.data.tsx

```diff
@@ -125,12 +125,8 @@ export function MustUploadFirstAlert(): JSX.Element {
       status='error'
       aria-live='polite'
       aria-atomic='true'
-      message={
-        <FormParagraph>
-          File upload and validation checks must be completed to save and
-          continue.
-        </FormParagraph>
-      }
+      headingLevel=''
+      message='Your file must be successfully uploaded and validation checks must be performed to continue to the next step.'
```

Comment:
@shindigira - There was a change here that I'm not sure was caught. We removed the second "must be" from this message. 

'Your file must be successfully uploaded and validation checks performed to continue to the next step.'